tool/plugin/plugin.iml

```diff
@@ -11,7 +11,5 @@
     </content>
     <orderEntry type="inheritedJdk" />
     <orderEntry type="sourceFolder" forTests="false" />
-    <orderEntry type="library" name="Dart SDK" level="project" />
```

Comment:
I don't know if these two lines are required. This is a Dart module so it makes sense to have them. Might be safest to leave them.

Reply:
Reverted.

IntelliJ removes it for me.
I have Dart plugin installed, and I use `2017.2` version of IntelliJ.
So, I don't know why it remove these items.
Do I to do something else than specified in https://github.com/scheglov/flutter-intellij/blob/master/CONTRIBUTING.md?

Reply:
`2017.2` or `2017.3`?

Reply:
I have `2017.2.6` installed.
```
IntelliJ IDEA 2017.2.6
Build #IC-172.4574.11, built on November 13, 2017
JRE: 1.8.0_152-release-915-b12 x86_64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Mac OS X 10.13.2
```